project/dexter-daemon/src/java/com/samsung/sec/dexter/daemon/job/MonitorForPlatzKeywordFile.java

```diff
@@ -0,0 +1,106 @@
+package com.samsung.sec.dexter.daemon.job;
+
+
+import java.io.File;
+import org.eclipse.core.runtime.IProgressMonitor;
+import org.eclipse.core.runtime.IStatus;
+import org.eclipse.core.runtime.NullProgressMonitor;
+import org.eclipse.core.runtime.Status;
+import org.eclipse.core.runtime.jobs.Job;
+import com.samsung.sec.dexter.core.config.DexterConfig;
+import com.samsung.sec.dexter.core.config.IDexterHomeListener;
+import com.samsung.sec.dexter.core.config.PlatzKeywordFile;
+import com.samsung.sec.dexter.core.exception.DexterRuntimeException;
+import com.samsung.sec.dexter.core.util.DexterClient;
+import com.samsung.sec.dexter.core.util.DexterUtil;
+import com.samsung.sec.dexter.daemon.DexterDaemonActivator;
+import com.samsung.sec.dexter.eclipse.ui.util.EclipseUtil;
+import com.samsung.sec.dexter.eclipse.ui.view.PlatzView;
+import org.eclipse.ui.progress.UIJob;
+
+public class MonitorForPlatzKeywordFile extends Job implements IDexterHomeListener{
+	public static long LAST_CONF_CHANAGED_TIME = -1;
+	private final static int MONITOR_DELAY = 500;
+	private File keywordFile;
+	
+	public MonitorForPlatzKeywordFile(){
+		super("Monitoring target keyword in sourceinsight");
```

Comment:
If developers are not in VD, this job should not be created.
Please check the existenance of PLATZ first.
(if you cannot change it in ver. 0.9.4, you can do it next version)